src/api/layers/layers-external.api.js

```diff
@@ -4,23 +4,47 @@ import WMTSCapabilitiesParser from '@/api/layers/WMTSCapabilitiesParser.class'
 import log from '@/utils/logging'
 
 /** Timeout for accessing external server in [ms] */
-const externalServerTimeout = 30000
+const EXTERNAL_SERVER_TIMEOUT = 30000
```

Comment:
I'm surprised you can import that in the `scripts/check-external-layers-providers.js` without adding a `export ...` before the constant declaration

Reply:
You are right but it works strangely. Anyway I'll add the `export`

Reply:
Done